test/CMakeLists.txt

```diff
@@ -760,41 +760,6 @@ if(${MIOPEN_TEST_WITH_MIOPENDRIVER})
     )
 endif()
 
-# ./bin/MIOpenDriver conv -n 128 -c 1024 -H 14 -W 14 -k 2048 -y 1 -x 1 -p 0 -q 0 -u 2 -v 2 -l 1 -j 1 -m conv -g 1 -F 1 -t 1
-# MIOPEN_DEBUG_CONV_IMMED_FALLBACK=0
-if(MIOPEN_EMBED_DB)
```

Comment:
@JehandadKhan [Notice] just in case - it seems that this code is working, but actually it should be `if(NOT MIOPEN_EMBED_DB STREQUAL "")` here.

Reply:
https://github.com/ROCmSoftwarePlatform/MIOpen/blob/29afc6d0d4d98a2e821a420ac3613d74d51b9b9b/test/gtest/conv_embed_db.cpp#L105

> @JehandadKhan [Notice] just in case - it seems that this code is working, but actually it should be `if(NOT MIOPEN_EMBED_DB STREQUAL "")` here.

@atamazov , I have been thinking if this Macro would be necessary

https://github.com/ROCmSoftwarePlatform/MIOpen/blob/29afc6d0d4d98a2e821a420ac3613d74d51b9b9b/test/gtest/conv_embed_db.cpp#L105